### PR TITLE
Truncate commit message on branches and prebuilds

### PR DESCRIPTION
**Pull request by gtsiolis**
## Description

This will truncate the commit message on branches and prebuilds.

## Related Issue(s)

Fix https://github.com/gitpod-io/gitpod/issues/5335

## Screenshots

| BEFORE | AFTER |
|-|-|
| Screenshot 2021-10-06 at 6 39 42 PM (2) | Screenshot 2021-10-06 at 6 39 55 PM (2) |

## How to test

1. Add a project with a long commit message on the default branch
2. Trigger a prebuild for the default branch
3. Check out the branches and prebuilds pages for that project

## Release Notes

```release-note
Truncate commit message on branches and prebuilds
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **gtsiolis** after the PR has been reviewed.

Associated issue: *#5335*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: components/dashboard/src/projects/Prebuilds.tsx [hunk 1]
```diff
@@ -179,8 +179,8 @@ export default function () {
                         </a>
                     </ItemField>
                     <ItemField className="flex items-center">
-                        <div>
-                            <div className="text-base text-gray-500 dark:text-gray-50 font-medium mb-1">{shortCommitMessage(p.info.changeTitle)}</div>
+                        <div className="truncate">
+                            <div className="text-base text-gray-500 dark:text-gray-50 font-medium mb-1 truncate">{shortCommitMessage(p.info.changeTitle)}</div>
```
Comment (gtsiolis) [inline components/dashboard/src/projects/Prebuilds.tsx:183]: **thought**: Not sure if I'm missing something here but this works better when using the `truncate` class on both elements. 🤷

**Comment by gtsiolis** [comment]
@geropl could you take a look at this small change when you're free?

**priority**: Low 🔅

**Comment by geropl** [comment]
@gtsiolis Sorry for this taking so long. I will try to improve my review workflow with the next free slot I can find :bow:

**Pull request merged**